assets/src/console/pages/payments/DetailPage.tsx

```diff
@@ -78,7 +93,9 @@ const DetailPage: React.FC<Props> = ({ service, gatewayAccount, payment }) => {
 
       <MenuTitle>
         <PageTitle title="Transaction detail" />
-        <Link to={`${location.pathname}/refund`}>Refund payment</Link>
+        {paymentStatusLabel(status) === "Success" && (
```

Comment:
you should use the `PaymentStatus` enum to compare with status, rather than comparing with the label, e.g. https://github.com/govau/pay/blob/bd6499c5c697e888db4d75a15621e412df2f30aa/assets/src/checkout/PaymentRoutes.tsx#L78